README.md

```diff
@@ -26,7 +24,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        androidTestCompile 'com.novoda:rxpresso:0.1.5'
+        androidTestCompile 'com.novoda:rxpresso:0.2.0'
```

Comment:
nit: this should be done after the PR has been merged in and after the artefact has been uploaded.